File: InstantSearchApp/japp/src/main/java/com/woowrale/instantsearch/japp/ui/search/local/LocalSearchView.java
package com.woowrale.instantsearch.japp.ui.search.local;

import com.woowrale.instantsearch.japp.data.model.Contact;
import com.woowrale.instantsearch.japp.ui.adapters.ContactsAdapterFilterable;
import com.woowrale.instantsearch.japp.ui.base.BasePresenter;

import java.util.List;

import io.reactivex.disposables.CompositeDisposable;

public interface LocalSearchView extends BasePresenter.View {

    CompositeDisposable getDisposable();

    ContactsAdapterFilterable getmAdapter();

    List<Contact> getContactsList();

}

File: InstantSearchApp/kapp/src/main/java/com/woowrale/instantsearch/kapp/ui/search/local/LocalSearchView.kt
package com.woowrale.instantsearch.kapp.ui.search.local

import com.woowrale.instantsearch.kapp.data.model.Contact
import com.woowrale.instantsearch.kapp.ui.adapters.ContactsAdapterFilterable
import com.woowrale.instantsearch.kapp.ui.base.BasePresenter
import io.reactivex.disposables.CompositeDisposable

interface LocalSearchView : BasePresenter.View {

    val disposable: CompositeDisposable

    fun setContactsList(contactList: List<Contact>)

    fun getContactsList():List<Contact>

    fun getmAdapter(): ContactsAdapterFilterable

}